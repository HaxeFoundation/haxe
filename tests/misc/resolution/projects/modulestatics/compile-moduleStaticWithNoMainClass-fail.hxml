-main ModuleStaticWithNoMainClass
-js moduleStaticWithNoMainClass.js
