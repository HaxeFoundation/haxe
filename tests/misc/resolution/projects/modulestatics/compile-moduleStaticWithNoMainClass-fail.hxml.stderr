ModuleStaticWithNoMainClass.hx:3: characters 15-23 : Module RootMod1 does not define type RootMod1